_posts/en/2020-08-12-better-tech-through-microprocurement-p1.md

```diff
@@ -0,0 +1,116 @@
+In 2016, the total spent by the GC on goods and services was ~$16 billion.
+From an IT point of view, this spending can include big-ticket items like new data centres and new national applications costing tens of millions of dollars, but can also include low dollar value items like paying someone to build a small prototype over the course of a few days. This is part one of a series looking at how micro-procurements can bring IT and procurement folks together to enable agile, rapid, and scalable digital work."
+---
+<!--markdownlint-disable MD033-->
+This is part one of a two part series looking at micro-procurement for digital work.
+In this first post, we'll look at: some context, the connection between IT transformation and procurement, as well as how, in addition to helping us deploy faster, micro-procurement could increase open source uptake in the GC, improve diversity in GC IT contracting and help GC IT employees acquire new skills by bringing in modern tools and practices.
+
+***
+
+The Government of Canada (GC) buys a lot of stuff.
+In 2016, the total spent by the GC on goods and services was ≈$16 billion.
+From an IT point of view, this spending can include big-ticket items like new data centres and new national applications costing tens of millions of dollars, but can also include low dollar value items like paying someone to build a small prototype or hiring a consultant to complete a gap analysis.
+
+To ensure that purchasing is working well across the GC, procurement teams are working hard on modernization efforts including the
+development of an [electronic procurement system](https://buyandsell.gc.ca/initiatives-and-programs/the-electronic-procurement-solution/about), enabling [agile procurement](https://www.tpsgc-pwgsc.gc.ca/app-acq/ma-bb/appagile-proagile-eng.html), [social procurement pilots](https://buyandsell.gc.ca/procurement-data/tender-notice/PW-18-00828087) and [contract language simplification](https://www.tpsgc-pwgsc.gc.ca/app-acq/ma-bb/simpcont-simpcont-eng.html).
+Interestingly, Low Dollar Value (LDV) procurements don't seem to make the list for procurement modernization priorities.
+This could be because these procurements are low risk and generally don't make headline news.
+However, for this same reason they provide the perfect opportunity for procurement and IT experimentation.
+
+<p class="tab">
+  Imagine that you manage an IT product and a team member notices a bug in one of the open source components, but doesn't have time to fix it.
+  What if you could post an opportunity to fix that bug for $2K, select someone to do the work and receive the fixed and tested code within a matter of days?
+</p>
+
+For the GC, Low Dollar Value is defined within PSPC's [Supply Manual](https://buyandsell.gc.ca/policy-and-guidelines/supply-manual/section/3/60) as contracts where the value is $25K or less for goods or $40K or less for services.
+Most of the spending on goods and services in the GC is on LDV items.
```

Comment:
I'm thinking the syntax may be a bit confusing: 
- Majority of contracts are on LDV
- Majority of money spent is on the big contracts.

By saying "_Most of the spending_", I feel the syntax seems to indicate that most of the money spent is on LDV?

Should we go for something a bit more towards: "_The vast majority of contracts awarded are_"

Reply:
Excellent catch!  Changing to your suggested wording now. :)

Reply:
Actually, now that I've changed it, that sentence seems somewhat unnecessary as the next sentence basically says the same thing. I'm thinking of deleting.

Reply:
Ok, I'm leaving it in.  It looks and sounds less duplicative in the paragraph than it looks in the code. :)